frontend/src/components/common/checkboxes/CheckBoxes.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect } from "react";
+import { Flex, VStack, Box, Center } from "@chakra-ui/react";
+
+import CheckBox, { CheckBoxStyleParams } from "./CheckBox";
+
+interface CheckBoxesParams extends CheckBoxStyleParams {
+  setStatus: (statuses: boolean[]) => void;
+  statuses: boolean[];
+  multiSelect?: boolean;
+  padding?: number;
+  borderStyling?: (index: number) => string | undefined;
+  backgroundStyling?: (index: number) => string | undefined;
+  children: React.ReactElement[];
+}
+
+const CheckBoxes = ({
```

Comment:
Broadly speaking, what is the benefit of a custom checkbox component over using checkboxes and radio buttons? Not opposed to it either way, but curious. Also if we do that, maybe we should rename this something more descriptive like `CheckBoxGroup` or something.

Reply:
More flexibility, but I've changed the implementation because it's probably better you use the libraries.